src/components/OSCALLoaderForm.js

```diff
@@ -13,11 +13,18 @@ const OSCALDocumentForm = styled("form")(
 );
 
 export default function OSCALLoaderForm(props) {
+  const url = useRef(null);
```

Comment:
I am wondering if we actually need this `useRef`. I added it as a wild guess. But I am curious if we can remove it?

Reply:
After playing around with removing it a bit, it seems that `useRef` may be needed. At the very least I am having trouble getting it to work without it.